components/logicbutton/stories/template.js

```diff
@@ -24,3 +24,11 @@ export const Template = ({
 			: undefined}
 	</button>
 `;
+
+export const DisabledTemplate = (args, context) => html`
```

Comment:
This template name feels more related to where it's used than what the content it contains is.  Can we call this something like "VariantCombo" instead so if it has other uses, we don't have to rename it.